framework/src/org/checkerframework/common/util/report/ReportAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,26 @@
+package org.checkerframework.common.util.report;
```

Comment:
I tried creating a clone of the Unqualified as ReportUnqualified. Using this clone, there's errors when running report-tests.

We can investigate why in the future. I'll keep it as Unqualified for now.